src/pip/_internal/wheel.py

```diff
@@ -1055,6 +1058,10 @@ def build(
                 )
                 if wheel_file:
                     build_success.append(req)
+
+                    self.wheel_filenames.append(
+                        str(os.path.relpath(wheel_file, output_dir))
```

Comment:
> If this is for mypy, I'd prefer to drop the type annotation and revisit this later.

Or actually, if mypy is flagging this, it might be worth taking a look into why it is flagging this. :)

Reply:
Yes, it was for mypy. So I'll try ``self.wheel_filenames = []  # type: List[bytes]``.

Reply:
Looking at https://github.com/python/typeshed/blob/master/stdlib/3/os/path.pyi#L149-L152, it'll make more sense for it to be `Union[bytes, Text]`.